src/main/kotlin/com/example/service/GeoIpService.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.service
+
+import com.maxmind.geoip2.DatabaseReader
+import com.maxmind.geoip2.exception.AddressNotFoundException
+import java.io.InputStream
+import java.net.InetAddress
+
+class GeoIpService {
```

Comment:
Please combine geoipservice and getcountryforipusecase into one file - there's no need to have two.